src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorController.ts

```diff
@@ -503,6 +503,12 @@ export class InteractiveEditorController implements IEditorContribution {
 		}
 	}
 
+	createSnapshot(): void {
+		if (this._activeSession?.lastExchange?.response instanceof EditResponse && this._strategy) {
+			this._activeSession.createSnapshot();
+		}
```

Comment:
FYI - I relaxed this check to always snapshot when the model has mutated. This allows for a sequence like 'edit-response', 'cell execute', 'markdown-response' response which I believe is wanted